src/Test/MakerTestCase.php

```diff
@@ -12,11 +12,16 @@
 namespace Symfony\Bundle\MakerBundle\Test;
 
 use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\MakerBundle\Maker\MakeEntity;
 
 class MakerTestCase extends TestCase
 {
     protected function executeMakerCommand(MakerTestDetails $testDetails)
     {
+        if ($testDetails->getMaker() instanceof MakeEntity && version_compare(PHP_VERSION, '7.1.0') < 0) {
```

Comment:
This MakerTestCase is a reusable one for any maker (including third-party ones). Hardcoding a special case for MakeEntity looks weird to me. I suggest adding a way to mark the test as skipped in MakerTestDetails instead.